Tests/MongoSwiftTests/MongoClientTests.swift

```diff
@@ -7,8 +7,34 @@ import XCTest
 final class MongoClientTests: MongoSwiftTestCase {
     func testListDatabases() throws {
         let client = try MongoClient()
-        let databases = try client.listDatabases(options: ListDatabasesOptions(nameOnly: true))
-        expect((Array(databases) as [Document]).count).to(beGreaterThan(0))
+
+        let databases = [
+            "db1",
+            "empty",
+            "db3"
+        ]
+
+        databases.forEach {
+            try? client.db($0).drop()
+            _ = try? client.db($0).createCollection("c")
```

Comment:
will this line ever fail?

Reply:
it might. I originally ignored the errors in case of any weirdness with mongoses, but I guess it makes more sense to just run the commands and fail on any errors. it seems preferable to failing in one of the later assertions and making debugging harder.